lib/rspec/core/formatters/exception_presenter.rb

```diff
@@ -41,38 +41,32 @@ def formatted_backtrace(exception=@exception)
             formatted_cause(exception)
         end
 
-        if RSpec::Support::RubyFeatures.supports_exception_cause?
```

Comment:
Please leave this branch, we prefer to shortcut the work / make life easier to delete when we update Ruby